app/display.py

```diff
@@ -0,0 +1,18 @@
+from app.book import Book
+from abc import ABC, abstractmethod
```

Comment:
Order and group your imports to follow PEP-8 here and in the other files as well